### PR TITLE
Fix version detection for builder build script

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

In order to only echo the build VERSION, some commands need their output redirected to stderr.

Fixes a regression introduced in #9604

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by brianmcarey — COMMENTED** [review]
/lgtm

/cc @xpivarc

**Comment by rmohr** [comment]
/approve

thanks for reviewing.

File: hack/builder/build.sh [hunk 1]
```diff
@@ -35,8 +35,8 @@ for ARCH in ${ARCHITECTURES}; do
         bazel_arch=${ARCH}
         ;;
     esac
-    ${KUBEVIRT_CRI} pull --platform="linux/${ARCH}" quay.io/centos/centos:stream9
-    ${KUBEVIRT_CRI} build --platform="linux/${ARCH}" -t "quay.io/kubevirt/builder:${VERSION}-${ARCH}" --build-arg SONOBUOY_ARCH=${sonobuoy_arch} --build-arg BAZEL_ARCH=${bazel_arch} -f "${SCRIPT_DIR}/Dockerfile" "${SCRIPT_DIR}"
+    ${KUBEVIRT_CRI} >&2 pull --platform="linux/${ARCH}" quay.io/centos/centos:stream9
```
[the remark this answers is not in the document]
Comment (rmohr) [reply hack/builder/build.sh:38]: Yep fmt did that. Is this a valid syntax?

**Comment by rmohr** [comment]
/unhold

the formatting seems to be valid.

**Pull request merged**